{"name": "Проверка получения списка заказов с неправильным трек номером", "status": "passed", "description": "Негативный сценарий проверка получения списка заказов по трек номеру {negative_data}", "parameters": [{"name": "negative_data", "value": "['', 400, 'Недостаточно данных для поиска']"}], "start": 1757920956770, "stop": 1757920957137, "uuid": "f527ed18-3c90-464e-b112-cf9a1be67a59", "historyId": "dbb9eafb477613e7e0d8ec85e9b2edac", "testCaseId": "20c349e9cae5542ad8901f9d5aca8ab0", "fullName": "tests.test_get_order_by_number.TestGetOrderByNumber#test_get_order_by_number_negative_check_response", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_by_number"}, {"name": "subSuite", "value": "TestGetOrderByNumber"}, {"name": "host", "value": "RUMS01CW-F915A8"}, {"name": "thread", "value": "12672-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_by_number"}], "titlePath": ["tests", "test_get_order_by_number.py", "TestGetOrderByNumber"]}